tests/atomic_fence/atomic_fence.cpp

```diff
@@ -0,0 +1,327 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
```

Comment:
There is some license header to be added.

Reply:
License header added.